crates/core_simd/tests/ops_impl/float_macros.rs

```diff
@@ -0,0 +1,237 @@
+macro_rules! float_tests {
+    { $vector:ident, $scalar:ident } => {
+        #[cfg(test)]
+        mod $vector {
+            use super::*;
+            use helpers::lanewise::*;
+
+            // TODO change to read from slice
```

Comment:
What does this mean?

Reply:
Hm.  Could clear up this comment.  Presumably in the future there will be a function that reads vectors from slices.

Reply:
Ah, much clearer!